.github/workflows/arm64.yml

```diff
@@ -83,6 +86,14 @@ jobs:
             --ignore=tests/scenarios/test_sync.py \
             --ignore=tests/scenarios/test_transformation.py \
             -k "not test_search"
+      - run: echo "TEST_FAILURE=true" >> $GITHUB_ENV
+        if: failure()
+      - name: Create JIRA issue
+        if: always()
+        run: |
+          pip install -r ./scripts/ci/requirements.txt
+          ./scripts/ci/create_jira_issue.py
+        shell: bash
```

Comment:
Is this really needed?

Reply:
No. I removed it.

Ref. https://github.com/Alfresco/acs-packaging/pull/2450